tests/test_revoke_authority_materialize_523.py

```
@@ -0,0 +1,512 @@
+        [{"dossier_id": revoke_dossier["id"], "decision": "promulgated"}],
+        content=content,
+    )
+    # 目标终结；initiative 停 tick
+    assert db.get_decree_dossier(target_id)["status"] == "closed"
+    assert db.conn.execute(
+        "SELECT status FROM issues WHERE id=?", (issue_id,),
+    ).fetchone()["status"] == "dropped"
+    # 旧案卷行仍在（非 undo 删除）
+    assert db.get_decree_dossier(target_id) is not None
+    assert db.get_decree_dossier(target_id)["decree_text"] == old_text
+    # 毁约走 0056 轨（皇威代价）
```

Ruff (0.16.1)

[warning] Comment contains ambiguous `（` (FULLWIDTH LEFT PARENTHESIS). Did you mean `(` (LEFT PARENTHESIS)?

(RUF003)